test/functional/bundleadd/add-install-time.bats

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/env bats
+
+load "../testlib"
+
+test_setup() {
+
+	create_test_environment "$TEST_NAME"
+	create_bundle -n bundle1 -f /file1,/file2 "$TEST_NAME"
+
+}
+
+test_teardown() {
+
+	destroy_test_environment "$TEST_NAME"
+
+}
+
+@test "Add a bundle showing detailed installation time" {
+
+	run sudo sh -c "$SWUPD bundle-add $SWUPD_OPTS bundle1 -t"
+
+	assert_status_is 0
+
+	expected_output=$(cat <<-EOM
+		Starting download of remaining update content. This may take a while...
+		Finishing download of update content...
+		Installing bundle\(s\) files...
+		Calling post-update helper scripts.
+		Raw elapsed time stats:
+			.* ms: Add bundles and recurse
+			.* ms: Consolidate files from bundles
+			.* ms: Download packs
+			.* ms: Download missing files
+			.* ms: Installing bundle\(s\) files onto filesystem
+		CPU process time stats:
+			.* ms: Add bundles and recurse
+			.* ms: Consolidate files from bundles
+			.* ms: Download packs
+			.* ms: Download missing files
+			.* ms: Installing bundle\(s\) files onto filesystem
+		Successfully installed 1 bundle
+		Raw elapsed time stats:
+			.* ms: Prepend bundles to list
+			.* ms: Install bundles
+		CPU process time stats:
+			.* ms: Prepend bundles to list
+			.* ms: Install bundles
+	EOM
+	)
+
+	assert_regex_in_output "$expected_output"
```

Comment:
I would use _assert_regex_is_output_ instead of _assert_regex_in_output_, the difference between _is_ and _in_ is that _is_ expects the command output to match what you specified in your test, and nothing more, while using _in_ you are only verifying a partial output. So it is better to use _assert_regex_is_output_ when possible.